cfme/common/vm.py

```diff
@@ -184,7 +271,7 @@ def delete(self, cancel=False, from_details=False):
                                                    handle_alert=not cancel)
         else:
             view = navigate_to(self.parent, 'All')
-            self.find_quadicon().ensure_checked()
+            self.find_quadicon().check()
```

Comment:
Why change this back to only a call to `check`?

Reply:
I didn't make this change. Looks like another commit was merged while this PR was waiting for review. I can rebase it.